src/Command/StatusCommand.php

```diff
@@ -103,6 +110,18 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
             'dry-run' => $args->getOption('dry-run'),
         ]);
         $manager = $factory->createManager($io);
+
+        if ($clean) {
+            $removed = $manager->cleanupMissingMigrations();
+            if ($removed === 0) {
+                $io->out('<info>No missing migrations to clean up.</info>');
+            } else {
+                $io->out(sprintf('<info>Removed %d missing migration(s) from the phinxlog table.</info>', $removed));
```

Comment:
```suggestion
                $io->out(sprintf('<info>Removed %d missing migration(s) from migration log.</info>', $removed));
```

Lets not use the table name, as it will be wrong for plugins.